.github/workflows/auto-translate.yml

```
@@ -109,4 +103,4 @@ jobs:
         with:
           commit_message: "🌐 Auto-translated content"
           branch: ${{ github.event.pull_request.head.ref }}
-          token: ${{ steps.generate-token.outputs.token }}
\ No newline at end of file
+          token: ${{ secrets.GITHUB_TOKEN }}
```

YAMLlint (1.37.1)

[error] no new line character at the end of file

(new-line-at-end-of-file)